spec/controllers/users_controller_spec.rb

```diff
@@ -113,6 +113,7 @@
     end
 
     before(:each) do
+      FactoryBot.create(:product_user, user: user, product: instruments.last)
```

Comment:
I'd rather not add to these older controller specs, I prefer system specs instead.  Could you add a spec to `user_access_list_spec.rb` to show the new column is present?